config/global.ini.php

```diff
@@ -835,6 +835,12 @@
 ; If set to 1, when rearchiving reports in the past we do not rearchive segment data with those reports. Default is 0.
 rearchive_reports_in_past_exclude_segments = 0
 
+; Enable HTTP checks on install or Diagnostics for required and recommended private directories.
+; Set this to 0 if you need to skip it because your hosting provider makes your site inaccessible
+; for some times if it is run, e.g. on install or when running diagnostics.
+; Default is 1.
+enable_required_directories_diagnostic = 1
```

Comment:
see https://github.com/matomo-org/matomo/pull/18014#discussion_r710674258 this is not executed on "install"

Reply:
@geekdenz can we add back the other removed valuable information from the previous comment like 

`http checks ...for required and recommended private directories.`
 and 
`; Set this to 0 if you need to skip it because your hosting provider makes your site inaccessible`

The config doesn't disable all HTTP checks, that's what the internet enabled config is there for. And we usually give users an idea when the setting may need to be disabled and how they can do this. So be good if we add back this information.

Reply:
@geekdenz this comment was resolved but we still need to tweak the description see previous comments. There's a needs review label on it but PR is also in status `Draft` so not sure if you still work on it or if it's ready for review.

Reply:
Meant to review this earlier today. But only got around to it now. Will try to keep in mind to remove the label also.